packages/mdc-slider/_mixins.scss

```diff
@@ -262,6 +262,19 @@
       height: 2px;
       overflow: hidden;
     }
+
+    &::after {
+      @include feature-targeting-mixins.targets($feat-structure) {
+        position: absolute;
+        top: 0;
+        left: 0;
+        display: block;
+        width: 100%;
+        height: 100%;
+        opacity: $opacity;
```

Comment:
Seems like build is failing because of undefined `$opacity`.

Should this property moved back to rail-color_ mixin?

Reply:
ops, yeah I was being too quick there. It's fixed now.